tools/viewers/viewers.py

```diff
@@ -0,0 +1,24 @@
+import click
+
+from viewers.yolo.click_yolo_viewer import yolo
+
+
+@click.group()
+def viewers():
+    """
+    Label Viewers
+
```

Comment:
```suggestion
\b
```

Similar to above. With the `\b` the `--help` command will format the docstring the same way it appears in the text editor.